docs/user/filesystem/commandline.rst

```diff
@@ -0,0 +1,175 @@
+      -p 's3a://mybucket/datastores/test' \
+      -e parquet \
+      --partition-scheme daily,z2-2bits \
+      -s s3a://mybucket/schemas/my-config.conf \
+      -C s3a://mybucket/schemas/my-config.conf \
+      --temp-dir hdfs://namenode:port/tmp/gm/1 \
+      --num-reducers 20 \
+      's3a://mybucket/data/2016/*'
+
+
+After ingest we expect to see a file structure with metadata and parquet files in S3 for our type named "myfeature"::
+
+    aws s3 ls --recursive s3://mybucket/datastores/test
+
+    datastores/test/myfeature/schema.sft
+    datastores/test/myfeature/metadata
+    datastores/test/myfeature/2016/01/01/0/0000.parquet
+    datastores/test/myfeature/2016/01/01/2/0000.parquet
+    datastores/test/myfeature/2016/01/01/3/0000.parquet
+    datastores/test/myfeature/2016/01/02/0/0000.parquet
+    datastores/test/myfeature/2016/01/02/1/0000.parquet
+    datastores/test/myfeature/2016/01/02/3/0000.parquet
+
+Two metadata files (``schema.sft`` and ``metadata``) store information about the schema, partition scheme, and list of
+files that have been created. Note that the list of created files allows the datastore to quickly compute available
+files to avoid possibly expensive directly listings against the filesystem. You may need to run update-metadata if you
+decide to insert new files.
+
+Notice that the bucket "directory structure" includes year, month, day and then a 0,1,2,3 representing a quadrant of the
+Z2 Space Filling Curve with 2bit resolution (i.e. 0 = lower left, 1 = lower right, 2 = upper left, 3 = upper right).
+Note that in our example Jan 1 and Jan 2 both do not have all four quadrants represented. This means that the input
+dataset for that day didn't have any data in that region of the world. If additional data were ingested the directory
+and a corresponding file would be created.
+
+update-metadata
+~~~~~~~~~~~~~~~
+
+Recompute the list of partitions stored within the metadata file in a filesystem datastore. This metadata file
+is used at query time in lieu of performing repeated directory listings.
+
+export
+~~~~~~
+
+Export GeoMesa features. Commonly used arguments to control export are:
+
+* ``-a`` - A comma-separated list of attributes and/or filter functions to export (e.g. geom,dtg,user_name)
+* ``-q``` - a GeoTools CQL query to select rows of data from the datastore
+* ``-F`` - an output format (e.g. csv, tsv, avro)
+* ``--query-threads`` - The number of threads to read files from the datastore
+* ``-m`` - Maxiumum number of features to export
+
+Example export commands::
+
+    $ geomesa-fs export  \
+      -p 's3a://mybucket/datastores/myds' \
+      -e parquet \
+      -f test_feature
+
+    $ geomesa-fs \
+      -p 's3a://mybucket/datastores/myds' \
+      -e parquet \
+      -f test_feature \
+      -F TSV \
+      -q "dtg >= '2016-01-02' and dtg < '2016-01-10' and bbox(geom, -5, -5, 50, 50)"
+
+version
+~~~~~~~
+
+Prints out the version, git branch, and commit ID that the tools were built with::
```

Comment:
Hmm...  looks like there's a mix of imperative and declarative.  I'd pick one and make it consist.  (E.g., either "prints, exports, ingests" or "print, export, ingest".)